flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -165,16 +165,19 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
 
       args.getIfNotNull<Boolean>("sendClientReports") { options.isSendClientReports = it }
 
-      args.getIfNotNull<Map<String, Any>>("sdk") { sdk ->
-        val name = sdk["name"] as? String
-        val version = sdk["version"] as? String
-        if (name != null && version != null) {
-          val sdkVersion = SdkVersion(name, version)
-          options.setSentryClientName("$name/$version")
-          options.setSdkVersion(sdkVersion)
-        }
+      val name = "sentry.java.android.flutter"
+      val version = io.sentry.android.core.BuildConfig.VERSION_NAME
```

Comment:
Question, should we use the Native SDK's version (Android in this case), or the Flutter SDK version?
I believe the Native SDK version would make more sense/easier here.